src/azul/chalice.py

```diff
@@ -193,33 +194,48 @@ def _api_gateway_context_middleware(self, event, get_response):
         finally:
             config.lambda_is_handling_api_gateway_request = False
 
-    hsts_max_age = 60 * 60 * 24 * 365 * 2
-
-    # Headers added to every response from the app, as well as canned 4XX and
-    # 5XX responses from API Gateway. Use of these headers addresses known
-    # security vulnerabilities.
-    #
-    security_headers = {
```

Comment:
Nitpick: I would refactor this from a variable to a method in a separate commit. This makes it easier to read the changes by isolating the noisy insertion of `()` away from the semantic changes.